script/cibuild

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+bundle exec jekyll build
```

Comment:
yes, good idea ... Should we also add the Gemfile.lock, so the build is reproducible?

Reply:
Yes, good idea. I will add it.